.github/workflows/release-linux.yml

```diff
@@ -10,37 +10,25 @@ on:
 
 jobs:
   build-wheel:
-    name: "Build ${{ matrix.build }} wheels on ${{ matrix.os }} ${{ matrix.arch }}"
-    runs-on: "${{ matrix.os }}-latest"
+    name: "Build ${{ matrix.build }} wheels on ${{ matrix.arch }}"
+    runs-on: "ubuntu-latest"
     strategy:
       matrix:
-        os: ["ubuntu"]
-        arch: ["x86_64"]
-        build: ["cp38-*", "cp39-*", "cp310-*", "cp311-*", "cp312-*", "pp38-*", "pp39-*"]
-        include:
+        arch: ["x86_64", "aarch64"]
+        build: ["cp38-*", "cp39-*", "cp310-*", "cp311-*", "cp312-*", "cp313-*", "pp38-*", "pp39-*"]
```

Comment:
```suggestion
        build: ["cp38-*", "cp39-*", "cp310-*", "cp311-*", "cp312-*", "cp313-*", "pp38-*", "pp39-*", "pp310-*"]
```

Reply:
we can add PyPy 3.10 and [soon PyPy 3.11](https://pypy.org/posts/2025/01/towards-pypy311-an-update.html)